assets/dev/scss/editor/_control-scrubbing.scss

```diff
@@ -0,0 +1,27 @@
+.e-ui-state--document-scrubbing-mode__on {
+
+	&, * {
+		cursor: ew-resize;
+	}
+
+	input.e-scrubbing--active {
+		&, &:focus {
+			box-shadow: 0 0 0 1px #{ darken( $editor-info, 20% ) } inset;
+		}
+	}
+
+	label.e-scrubbing--active {
+		color: #{ darken( $editor-info, 20% ) };
+		user-select: none;
+	}
+
+	// When scrubbing, enable scrubbing cursor effects also on the 'preview' iframe area.
+	iframe {
+		pointer-events: none;
+	}
+}
+
+label.e-scrubbing-over {
+	cursor: ew-resize;
+}
+
```

Comment:
Why it's not nested under the UI state class?

Reply:
It doesnt suppose to be inside. the label shows special cursor when its not 'scrubbing' state.